packages/calypso-e2e/src/lib/components/comments-component.ts

```diff
@@ -105,9 +105,8 @@ export class CommentsComponent {
 			// See the like() method for info on the following method call.
 			await waitForWPWidgetsIfNecessary( this.page );
 
-			const likeButtonFrame = this.page
-				.frameLocator( `iframe[name^="like-comment-frame"]:below(:text("${ comment }"))` )
-				.first();
+			const commentContent = this.page.locator( '.comment-content', { hasText: comment } );
+			const likeButtonFrame = commentContent.frameLocator( "iframe[name^='like-comment-frame']" );
```

Comment:
Just using some modern syntax here, not a crucial change.

Reply:
I had an experimental branch where I made a similar change, so I can vouch it works.